mimic/pretrained_model_factory.py

```diff
@@ -0,0 +1,95 @@
+"""Pretrained model factory."""
+from mimic.model.markov_chain_model import MarkovChainModel
+from mimic.model.lstm_model import LSTMModel
+from mimic.text_generator import TextGenerator
+import mimic.util as utils
+import os
+import logging
+
+
+class PretrainedModelFactory:
+    """
+    Pretrained Model Class.
+
+    Class that creates a text generator of a pretrained model chosen by user.
+    """
+
+    def __init__(self):
+        """Initialize a TextGenerator with the given pretrained model type."""
+        logging.basicConfig(format='%(asctime)s %(message)s',
```

Comment:
I don't think we need to set the logging config here again.

Reply:
removed